website/source/api/health.html.markdown

```diff
@@ -61,7 +61,8 @@ $ curl \
     "Notes": "",
     "Output": "",
     "ServiceID": "",
-    "ServiceName": ""
+    "ServiceName": "",
+    "ServiceTags": null
```

Comment:
ok, I'll add it there where we return tags. I'll do that as a separate commit since it'll affect more than just this one.